pkg/subctl/benchmark/latency.go

```diff
@@ -65,9 +71,8 @@ func StartLatencyTests(intraCluster bool) {
 func runLatencyTest(f *framework.Framework, testParams benchmarkTestParams) {
 	clusterAName := framework.TestContext.ClusterIDs[testParams.ClientCluster]
 	clusterBName := framework.TestContext.ClusterIDs[testParams.ServerCluster]
-	var connectionTimeout uint = 20
-	var connectionAttempts uint = 3
-	var netperfPort = 12865
+	var connectionTimeout uint = 5
```

Comment:
Using 5 seconds can cause timing issues on some platforms. The default seems to be 60 secs and connectionAttempts as 2. Can we re-use the defaults here?

Also, IMO we should support these arguments so that users can override them if required. WDYS?

Reply:
We can revert to use defaults.
Do you mean to allow users to supply those arguments from command-line? 
I think it's a good idea

Reply:
Yes, I was thinking it might be a good idea to allow users to override these params during runs (if required). But for now, IMO lets use defaults.

Reply:
currently, the code is not using connectionTimeout and connectionAttempts params:
https://github.com/submariner-io/shipyard/blob/6a023e910387a2bf9f17ad9255efb14439b66b37/test/e2e/framework/network_pods.go#L339

Do you think it's need to be added?

Reply:
Basically when running tests aganist Globalnet deployment, the initial connection might fail because of the way we are creating the connectorPods as it takes few secs for the globalIP annotation to be applied on the connectorPods. So, its more important in Globalnet deployments that we retry atleast twice (i.e., connectionAttempts = 2).

I see that we have disabled Latency tests for Globalnet deployments, so its fine for now. But in a future PR please consider supporting connectionTimeout/connectionAttempts when Latency tests are enabled for Globalnet.